rust/src/main.rs

```diff
@@ -11,14 +11,13 @@ use clap::Parser;
     about,
     long_about = "Nitor Vault, see https://github.com/nitorcreations/vault for usage examples"
 )] // Reads info from `Cargo.toml`
-
 struct Args {
     #[arg(short, long, help = "List available secrets")]
     all: bool,
 
     #[arg(
         long,
-        help = "Describe cloudformation stack params for current configuration"
+        help = "Describe Cloudformation stack params for current configuration"
```

Comment:
same as the previous, if we are to use proper capitalization it should be "CloudFormation", which imo is somewhat unnecessary, but am not opposed to it either.